javascript/ql/src/Statements/UselessRangeCheck.ql

```diff
@@ -0,0 +1,28 @@
+/**
+ * @name Useless range check
```

Comment:
What @esben-semmle says ^. Might be a good idea to rephrase.

Reply:
I don't believe there was a discussion? What happened was that I asked a question in that PR, which I now regret, because it was never answered, and instead led to yet more ways to rephrase the word "useless".

But there are many queries named "useless"-something and randomly renaming a subset of these isn't the answer. I believe there's been some effort towards unifying the tags and IDs of queries across languages, and renaming just the JS queries would be counterproductive to that.